LOCALIZATION.md

```diff
@@ -0,0 +1,82 @@
+# Docs Localization
+
+## Introduction
+
+Welcome to the [NEAR Dev Docs](https://docs.near.org/) translation project.
+This article is aimed to community members that want to help in the localization of NEAR's documentation.
+If you feel comfortable translating from English into your local language, you're more than welcome to contribute with this project.
+
+### Roles
+
+For simplicity, the translation project is hosted on [Crowdin](https://crowdin.com/project/near-docs) and anyone is free to register and join.
+There are 3 basic roles across the Crowdin project:
+
+1. **Translator:** translates documentation
+2. **Proof Reader:** helps proof reading localized documents
+3. **Manager:** manages memberships and helps with technical issues
+
+
+### How to become a Translator
+
+To become a certified NEAR docs translator, you need to pass a pre-translation test by working on a selected document translation. A proof reader will review your document, and if your translation proves to be high-quality, your application will be accepted and you'll be accepted as a NEAR Certified Translator.
+
+> To become a certified translator, please join our [Crowdin project](https://crowdin.com/project/near-docs) and [create a new topic](https://crowdin.com/project/near-docs/discussions) with your application.
+> The NEAR team will review your request, assign a test, and follow up with you as needed.
+>
+> If you have questions, please contact us on [Discord](https://near.chat). 
+
+## How to translate
+
+Before you start working on the translation of an article, you need to get pre-approval from the community first. The reason is to make sure the plans are transparent in the community, and the proof-readers can help to review and approve the translation once it's ready.
+
+You can check the current translation plans available for each language on the [Tasks section](https://crowdin.com/project/near-docs/tasks):
+
+![image](website/static/docs/assets/crowdin-tasks.png)
+
+If there's no active plan for your language, you can submit a proposal by opening a thread on the [Discussions section](https://crowdin.com/project/near-docs/discussions).
+Once your plan is pre-approved by the community, you can just go to the [Crowdin project](https://crwd.in/near-docs) and work on the translation of the specific markdown file.
+
+### Translation plan
+
+For each language, the current plan is to translate all the documents with the following priorities.
+
+**Website**
+
+1. `website` (except the `docusaurus-plugin-content-blog` folder)
+
+
+**Developer**
+
+1. `concepts`
+2. `develop`
+3. `tools`
+4. `api`
+5. `tutorials`
+6. `videos`
+
+
+**Community**
```

Comment:
And it seems that community chapter has been removed
```suggestion

```

Reply:
fixed